docs/user-guide/pod-security-policy/index.md

```diff
@@ -137,12 +137,12 @@ A pod must validate every field against the PSP.
 Here is an example Pod Security Policy. It has permissive settings for
 all fields
 
-{% include code.html language="yaml" file="psp.yaml" ghlink="/docs/user-guide/pod-security-policy/psp.yaml" %}
+{% include code.html language="json" file="psp.json" ghlink="/docs/user-guide/pod-security-policy/psp.json" %}
```

Comment:
the actual file is `.yaml` which is incorrect, can you rename it so it has the proper extension?